### ISSUE TITLE
V4 - Make passwords longer and stronger

### ISSUE THREAD

**Issue by rhbkweb** [EasyEngine/feature-requests#70]
Hi. Please make the default passwords that EE v4 generates stronger.

I notice that by default all passwords are created with 12 characters and without special characters. My suggestion would be that all passwords by default to have at least 26 characters, including special characters like #, $, %…

Hope EE team can consider this. Thanks

**Comment by michacassola** [comment]
If we are thinking security why not make them even longer?

**Comment by rhbkweb** [comment]
> If we are thinking security why not make them even longer?

Yes, they can be longer. I just mention 26 characters since is what I normally use. But as important as being longer, the passwords must have special characters. At the moment (v4.0.6) just has regular letters and numbers.

**Comment by hbcoding-web-design** [comment]
This would be a great addition to the feature request! Longer passwords was the first thing that I needed. I updated the WordPress password, however, it does NOT update on the command line when using "sudo ee site info example.com"

**Comment by Dudo1985** [comment]
I agree
